manager/plugin/plugin.go

```diff
@@ -45,8 +46,12 @@ type vatzPluginManager struct {
 	home string
 }
 
-func (m *vatzPluginManager) Init() error {
-	return initDB(fmt.Sprintf("%s/%s", m.home, pluginDBName))
+func (m *vatzPluginManager) Init(runType tp.Initializer) error {
+	dbName := pluginDBName
+	if runType == tp.TEST {
+		dbName = "vatz-test.db"
+	}
```

Comment:
This is simple and maybe works find but we could find a better way because if we use "if" statement for the testing, that means we are adding the new codes, which are not usable on real env and not testable.

If we can change `home` and `dbName` from the outside of the `plugin` module, I think we can make it work but it should be a little bit tricky due to `pluginDBName` is not accessible from the outside of the module.

Let's think a little bit more.

Reply:
yeah, that's what I thought. this is temp way to handle bug for the moment.